src/client/debug.go

```diff
@@ -19,14 +19,16 @@ func (c APIClient) Dump(w io.Writer) error {
 }
 
 // Profile writes a pprof profile for pachd to w.
-func (c APIClient) Profile(profile string, duration time.Duration, w io.Writer) error {
+func (c APIClient) Profile(profile string, duration time.Duration, workerIP string, sidecar bool, w io.Writer) error {
```

Comment:
I think we should expose this as a pod / container name rather than an ip and bool. It's a little more friendly that way. The container name can default to `user` so you don't need to type anything in the more common case.

Reply:
I could probably do that, but it would be a more involved change. This is building on top of the functionality we have for setting up connections to workers using the worker etcd entries. We could just convert from pod to IP then use the same functionality internally?

Reply:
Converting from pod to IP sounds like it should work. I think it's worth it to do this, otherwise we'll need docs on how to find the ip of the pod, which isn't too difficult but still will force users to go through an extra step whereas I think they probably already know how to get the pod names, and those are possible to get through the api, unlike the ip addresses.